tests/system/providers/google/cloud/bigtable/example_bigtable.py

```diff
@@ -63,24 +63,35 @@
 from airflow.utils.trigger_rule import TriggerRule
 from tests.system.providers.google import DEFAULT_GCP_SYSTEM_TEST_PROJECT_ID
 
-ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
+ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID").lower().replace('_', '-')
```

Comment:
this line is okay, you don't need to change the content of this variable itself. In every place this stirng is used you can add only 'replace()', without 'lower()'